IdentityCore/src/webview/response/MSIDSwitchBrowserResponse.m

```diff
@@ -74,6 +74,24 @@ - (BOOL)useV2WebResponseHandling
     return useV2WebResponseHandling;
 }
 
++ (BOOL)isMyUrl:(NSURL *)url operation:(NSString *)operation
```

Comment:
nit: please consider to make the name more specific to the flow :)

Reply:
Is this method covered through other UTs?

Reply:
yes, it is covered explicitly since it is used in initWithURL:redirectUri:context:error. See MSIDSwitchBrowserResponseTest and MSIDSwitchBrowserResumeResponseTest.